### DISCUSSION TITLE
Why I install 2.0.0-next.9 but got old v1 code installed

### DISCUSSION THREAD

**Discussion by Jokcy** [mdx-js/mdx#1700] [Q&A] [answered]
<img width="1191" alt="截屏2021-09-28 下午9 26 12" src="https://user-images.githubusercontent.com/2088642/135095810-e77a7df4-3073-4424-a235-f1431d369bc4.png">
You can see in this screenshot, the version in package.json is 2.0.0-next.9 but the actual files in node_modules still old. What should I do to install the v2 version?

**Comment by wooorm** [comment, marked as answer]
That does indeed look like the latest released code.
There is much more work done which you can see on github, but that isn't released yet.

**Reply by Jokcy** [reply]
So how can I start to use the next version of mdx, I do not want to begin with the old version then update my code to feat with the newer release after a couple of months.

**Reply by wooorm** [reply]
Use xdm
